test/controllers/update/overseas.entity.query.controller.spec.ts

```diff
@@ -88,5 +89,22 @@ describe("OVERSEAS ENTITY QUERY controller", () => {
       expect(resp.status).toEqual(500);
       expect(resp.text).toContain(SERVICE_UNAVAILABLE);
     });
+
+    test('renders the OVERSEAS_ENTITY_QUERY_PAGE page with both error messages', async () => {
+      const resp = await request(app)
+        .post(config.OVERSEAS_ENTITY_QUERY_URL)
+        .send({ oe_number: '' });
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(ErrorMessages.OE_QUERY_NAME);
+      expect(resp.text).toContain(ErrorMessages.INVALID_OE_LENGTH);
+    });
+
+    test('renders the OVERSEAS_ENTITY_QUERY_PAGE page with correct error message', async () => {
+      const resp = await request(app)
+        .post(config.OVERSEAS_ENTITY_QUERY_URL)
+        .send({ oe_number: '123' });
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(ErrorMessages.INVALID_OE_LENGTH);
```

Comment:
I would add a `.not.toContain(ErrorMessages.OE_QUERY_NAME)` here to specifically only check for the one message. Example of this elsewhere.